sdk/core/core-amqp/test/requestResponse.spec.ts

```diff
@@ -174,84 +160,4 @@ describe("RequestResponseLink", function() {
       assert.equal(expectedErrorRegex.test(err.message), true);
     }
   });
-
-  it("should abort a request and response correctly when retried", async function() {
```

Comment:
We should add a test case for when `sendRequest()` is called with an abort signal which has already been fired. This will cover the case being discussed at https://github.com/Azure/azure-sdk-for-js/pull/4322#discussion_r305167936

Reply:
Agreed

Reply:
Added this test